src/components/view/sources/sources.ts

```diff
@@ -0,0 +1,54 @@
+import './sources.css';
+
+
+export interface IDataSources {
+    status: string;
+    sources?: Array<Source>;
+    totalResults?: number;
+    articles?: Array<Text>;
+};
+
+export type Source = {
```

Comment:
Please use a more informative name. For example, NewsSource or something else

Reply:
Names changed to more informative.